packages/console/src/scss/_underscore.scss

```diff
@@ -22,7 +22,7 @@
 }
 
 @mixin form-text-field {
-  width: dim.$form-text-field-width;
+  width: 100%;
```

Comment:
just remove the width styling? most likely the default value will be the full width of the container.

Reply:
No, this style is used on other pages, before we replace them all, I think it's good to leave it here right now.
BTW, fields that have `display: inline-block` will need this style.
E.g. `<CopyToClipboard />`
<img width="512" alt="image" src="https://user-images.githubusercontent.com/10806653/202607362-993350c5-913c-41e2-8863-e87d371f1e48.png">

Reply:
Add a TODO or FIXME tag?